roles/aks-setup/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Create resource group digitransit-aks-{{ az_environment }}"
+  command: "az group create --name digitransit-aks-{{ az_environment }} --location westeurope"
+  delegate_to: localhost
+  ignore_errors: "{{ ansible_check_mode }}"
+
+- name: "Create digitransit-aks-{{ az_environment }}-cluster"
```

Comment:
I think -cluster could be left out of the name here and switch order of aks and env